dataPipelines/gc_scrapy/gc_scrapy/pipelines.py

```diff
@@ -329,29 +329,29 @@ def process_item(self, item, spider):
         if item.get("source_fqdn") is None:
             item["source_fqdn"] = get_fqdn_from_web_url(source_page_url)
 
-        if item.get("version_hash") is None:
+ #       if item.get("version_hash") is None:
             # ensure doc_name is part of hash
-            item["version_hash_raw_data"]["doc_name"] = item["doc_name"]
-            item["version_hash"] = dict_to_sha256_hex_digest(item["version_hash_raw_data"])
+            # item["version_hash_raw_data"]["doc_name"] = item["doc_name"]
+            # item["version_hash"] = dict_to_sha256_hex_digest(item["version_hash_raw_data"])
 
         if item.get("access_timestamp") is None:
-            item["access_timestamp"] = datetime.now().strftime("%Y-%m-%d %H:%M:%S.%f")
+            item["access_timestamp"] = datetime.now().strftime("%Y-%m-%dT%H:%M:%S.%f") # T added as delimiter between date and time
 
         if item.get("publication_date") is None:
-            item["publication_date"] = "N/A"
+            item["publication_date"] = None
```

Comment:
we really need to test this. "N/A" works in the pipeline, but None will potentially throw out errors